Tests/TuistBuildAcceptanceTests/BuildAcceptanceTests.swift

```diff
@@ -96,6 +96,14 @@ final class BuildAcceptanceTestFrameworkWithSwiftMacroIntegratedWithXcodeProjPri
     }
 }
 
+final class BuildAcceptanceTestMultiplatformAppWithExtensions: TuistAcceptanceTestCase {
+    func test() async throws {
+        try setUpFixture("multiplatform_app_with_extensions")
```

Comment:
typo here, the fixture is missing the `s`
```suggestion
        try setUpFixture("multiplatform_app_with_extension")
```